### DISCUSSION TITLE
cloud hubs domain stop working today and show blank page for home and admin

### DISCUSSION THREAD

**Discussion by nonew** [mozilla/hubs#3967] [Hubs Cloud Questions] [answered]
I am using AWS free tier and I noticed S3 request exceed monthly free limit. is that the reason?

I have configured content CDN by cloudflare. from Chrome dev console, there is severl errors, such as:

refused to load the stylesheet 'https://xxx-1-assets.xxx.com/hubs/assets/stylesheets/support-0943be745a85162a87bb.css' because it violates the following Content Security Policy directive: "style-src 'self' https://fonts.googleapis.com https://cdn.jsdelivr.net https://hubs-xxx-com-cors-proxy.virtualstore.workers.dev:443 https://hubs-xxx-com-proxy.virtualstore.workers.dev  'unsafe-inline'". Note that 'style-src-elem' was not explicitly set, so 'style-src' is used as a fallback.

**Comment by rawnsley** [comment, marked as answer]
The CSP error isn't likely to be related to your S3 quota being exceeded, but that is likely to cause you other problems.

I haven't configured a CDN myself on Hubs, but I expect you need to add the CDN domains to the CSP whitelist somehow. This is the list that the page is allowed to download content from to avoid injections of HTML that download malicious script files.

Check your CDN config in Hubs.

**Reply by nonew** [reply]
Thank you! Yes, you are right, the S3 quota doesn't matter.

The issue is fixed after I reboot my EC2 instance. Cloudflare CDN is working without change.

I cannot remove imported scenes and images even if UI prompts me they were removed. When I come back they are still there. Sometimes I will encounter a 404 not found error when removing them.

Is there anyone who experiences this and knows how to resolve this issue?

**Reply by rawnsley** [reply]
Glad you are up and running now.

Regarding your imported data problem: create a new GitHub issue with as much information as possible.

**Reply by nonew** [reply]
@rawnsley  Thanks for your response!

I created a new thread: https://github.com/mozilla/hubs/discussions/3974, but no one responded.
could you recommend whom should I @ for a reply?

**Comment by nonew** [comment]
cloud hubs domain stop working again today.

when I try to resolve CORS issue of my Hubs Cloud, I refer to page: https://github.com/mozilla/hubs/discussions/2890
I terminate my EC2 HC instance, another one automatically launch.
But I cannot open my domain https://{my domain}.com now,  it show my blank page.
Before that I disabled content CDN in admin panel of Hubs Cloud (https://{my domain}.com/admin.

Please help me out of here, I have urgent project based on it. 

I open Chrome Dev console and here is one of error logging:
Refused to load the stylesheet 'https://hubs-{my domain}-com-proxy.virtualstore.workers.dev/hubs/assets/stylesheets/support-0943be745a85162a87bb.css' because it violates the following Content Security Policy directive: "style-src 'self' https://fonts.googleapis.com https://cdn.jsdelivr.net https://{my domain}-1-cors-proxy.{my domain}.com:443 https://{my domain}-1-assets.{my domain}.com  'unsafe-inline'". Note that 'style-src-elem' was not explicitly set, so 'style-src' is used as a fallback.

Thanks!

**Comment by nonew** [comment]
@robinkwilson 
After I redeploy the custom client, my domain works again. thanks for your help.
